cluster/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM ocaml/opam:debian-ocaml-4.13 AS build
 RUN sudo apt-get update && sudo apt-get install libc6-dev libev-dev capnproto m4 pkg-config libsqlite3-dev libgmp-dev libffi-dev -y --no-install-recommends
 RUN opam remote add origin 'https://opam.ocaml.org' --all-switches && opam remote remove default && opam update
+RUN sudo ln -f /usr/bin/opam-2.1 /usr/bin/opam
 RUN opam install dune>=2.8 ppx_expect>=v0.14.1 prometheus ppx_sexp_conv dune-build-info lwt>=5.4.1 capnp-rpc-lwt>=1.2 capnp-rpc-net capnp-rpc-unix>=1.2 extunix>=0.3.2 logs conf-libev digestif>=0.8 fpath lwt-dllist prometheus-app=1.1 cohttp-lwt-unix sqlite3 psq mirage-crypto>=0.8.5 ppx_deriving ppx_deriving_yojson astring fmt>=0.8.9 cmdliner tar-unix>=2.0.0 yojson sexplib sha
-RUN mkdir src && cd src && git clone 'https://github.com/ocurrent/ocluster' && opam pin -yn ocluster
-RUN cd src && git clone 'https://github.com/ocurrent/obuilder' && opam pin -yn obuilder
+RUN mkdir src && cd src && git clone 'https://github.com/ocurrent/ocluster' && git clone 'https://github.com/ocurrent/obuilder' && opam pin -yn ocluster obuilder
```

Comment:
I'm running into some issues between obuilder and ocluster... It seems to work better with:
```
RUN mkdir src && cd src && git clone --recurse-submodules 'https://github.com/ocurrent/ocluster' && opam pin -yn ocluster
```